setup.py

```diff
@@ -3,7 +3,7 @@
 
 NAME = "looker_deployer"
 VERSION = pkg.__version__
-REQUIRES = ["looker-sdk==0.1.3b7", "oyaml", "python-json-logger"]
+REQUIRES = ["looker-sdk>=0.1.3b8", "oyaml", "python-json-logger"]
```

Comment:
I wonder if it's a good idea to target the latest version of the SDK. You must have thought through this since you changed the == to >= but since we've had breaking changes a few times with the beta SDKs this might just cause trouble.

Reply:
good thought - i fixed it.